packages/keycodes/README.md

```diff
@@ -0,0 +1,13 @@
+# @wordpress/keycodes
+
+Keycodes utilities for WordPress.
+
+## Installation
+
+Install the module
```

Comment:
I've seen this in a few other packages and it seems so blunt and unhelpful 😉

Can we say "Install with `npm`:"?

Reply:
I think we should do this in a separate PR to keep consistency between the packages. and yes feel free to add some docs :)

Reply:
Ah, fair point. I'll do that separately, but I'd like to add usage docs to this one 😄

Reply:
Filed #7579, thanks!